### DISCUSSION TITLE
Debian ?

### DISCUSSION THREAD

**Discussion by jbhrfx9280** [nickjj/docker-django-example#26] [Q&A]
It appears this example was built for Debian based platform. Can this be converted over to run on RHEL? I have tried and it fails. I have worked in many large data centers and the DOJ and none of them use Debian based Linux servers.  I would love to try some of these examples on the platform I know.

**Comment by nickjj** [comment]
The Dockerfile uses Debian but you can use whatever distro you want for your Docker host, there's nothing specific that would cause things not to work on a RHEL based system.

If you wanted to use RHEL inside of the running environment of the app you would need to change the Dockerfile to use RHEL instead of Debian and then re-write all of the bits that are specific to Debian.

**Comment by jbhrfx9280** [comment]
Thank you for responding so fast. I really want to follow you and learn
from all your examples and classes. You seem to be well known in the
industry and someone that knows what he's talking about. I appreciate your
time.
John

**Reply by nickjj** [reply]
No problem, happy to help!
